trees/PrescriptiveTree.py

```diff
@@ -26,7 +26,7 @@ class PrescriptiveTreeClassifier(ClassifierMixin, BaseEstimator):
     """
     Parameters
     ----------
-    depth : int
+    depth : intClassifierMixin
         A parameter specifying the depth of the tree
     time_limit : int
         The given time limit for solving the MIP in seconds
```

Comment:
Can you please update the attributes part of the docstring. We don't need to include all attributes of the class but perhaps some of the most important such as b, p, w, etc.